### DISCUSSION TITLE
install nextcloud AIO problem nextcloud office

### DISCUSSION THREAD

**Discussion by manulpz** [nextcloud/all-in-one#1281] [Questions] [answered]
I am installing with these commands the docker nextcloud AIO. everything starts fine for me but there is no way to get the collabora to work for me to read documents. do I have to do something special? Is there a way to test that the collabora container is working fine? soy newbie.

sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 80:80 \
--publish 8080:8080 \
--publish 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest

thanks

**Comment by szaimen** [comment, marked as answer]
Sorry, no idea. Works over here. Is maybe something in the container logs or Browser logs and did you enable Collabora in the AIo interface?
